Changelog.md

```diff
@@ -9,6 +9,7 @@ Features:
 Bugfixes:
  * Disallow unknown options in `solc`
  * Inline assembly: support the `address` opcode
+ * Inline assembly: disallow magic variables (`this`, `super`, etc.)
```

Comment:
Does this sound better: `magic variables (`this`, `super`, etc.) are not visible in inline assembly and they can be used as local variables names` ?